app/src/main/java/nl/tudelft/b_b_w/controller/BlockController.java

```diff
@@ -67,39 +67,10 @@ public final boolean blockExists(String owner, String key, boolean revoked) {
      * @inheritDoc
      */
     @Override
-    public final List<Block> addBlockToChain(Block block) {
```

Comment:
You should not delete this method. This method has more checks than the normal addblock method. Or you could change the old block exists method to the new one.